data/scenarios/Testing/1034-custom-attributes.yaml

```diff
@@ -0,0 +1,134 @@
+version: 1
+name: Rainbow color custom attributes
+description: |
+  Custom attributes with rainbow colors
+creative: false
+attrs:
+  - name: rainbow1
+    fg: ffadad
```

Comment:
We could probably pretty easily support both with and without a hash.

Reply:
I agree with supporting both.  On the one hand, a leading `#` now requires the string to be quoted, as otherwise it is interpreted as a YAML comment.  On the other hand, a leading `#` allows the color to be recognized by the [colorize extension](https://marketplace.visualstudio.com/items?itemName=kamikillerto.vscode-colorize) (when [configured to include `yaml` language](https://marketplace.visualstudio.com/items?itemName=kamikillerto.vscode-colorize#colorize.languages-array)):

![Screenshot from 2023-01-28 22-44-02](https://user-images.githubusercontent.com/261693/215309950-cd55b4d6-981c-46a6-8e43-e9c9cc08c6c1.png)